data/pgroonga.sql

```diff
@@ -434,6 +434,12 @@ CREATE FUNCTION pgroonga_wal_set_applied_position()
 	LANGUAGE C
 	STRICT;
 
+CREATE FUNCTION pgroonga_list_broken_indexes()
+	RETURNS void /* todo */
+	AS 'MODULE_PATHNAME', 'pgroonga_list_broken_indexes'
+	LANGUAGE C
+	STRICT
+	PARALLEL SAFE;
```

Comment:
Could you add the same SQL to `data/pgroonga--3.2.0--3.2.1.sql`?
Could you also add `DROP FUNCTION pgroonga_list_broken_indexes;` to `data/pgroonga--3.2.1--3.2.0.sql`?

Reply:
I was going to add it after it was fixed because there will always be changes, but should I add it already?

Reply:
Yes, please.
FYI: `data/pgroonga--3.2.1--3.2.0.sql` will not be changed.

Reply:
I got it!